templates/default/clients.conf.erb

```diff
@@ -6,9 +6,8 @@
 
 <% node['freeradius']['clients'].each do |client, client_hash| %>
 client <%= client %> {
-  ipaddr = <%= client_hash['ipaddr']  %>
-  netmask = <%= client_hash['netmask'] %>
-  secret = <%= client_hash['secret'] %>
-  nastype = <%= client_hash['nastype'] %>
+<% client_hash.each do |key, val| %>
```

Comment:
Actually current case isn't compatible with FreeRadius version 3, because for example `nastype` is changed in FreeRadius 3 to `nas_type` and so on :)